README.md

```diff
@@ -39,12 +47,19 @@ helm schema-gen install/kubernetes/cilium/values.yaml > install/kubernetes/ciliu
 ```
 
 Don't forget to commit the changes, if any.
-Once we are done, we can push our new branch to our fork and it will be ready to be used by `vendir`.
+Once we are done, we can push our new branch to our fork, and it will be ready to be used by `vendir` from here.
 
 Then, in this repository, we need to update the `vendir` configuration in `vendir.yml` to use the new branch we just pushed, and run the make target `APPLICATION=cilium make update-chart`.
 With the generated changes, let's create a new pull request so that everyone can review the changes that will be applied to the cilium chart.
-If we need further customizations, we can keep adding commits on the new branch on the fork, and re-run `make update-chart` to update the generated files.
+If we need further customizations, we can keep adding commits on the new branch on the fork, and re-run `APPLICATION=cilium make update-chart` here to update the generated files in the app.
+
+Once we are happy with the changes, we need to make the `main` branch in our fork to point to our latest commit
+```
+git branch -f main
```

Comment:
Maybe this is more explicit. Without specifying the last part, it will move `main` to werever `HEAD` is currently 
```shell
git branch -f main <new-release-branch>
```